### DISCUSSION TITLE
Unchecked checkboxes look checked in the docs

### DISCUSSION THREAD

**Discussion by WaylonWalker** [Textualize/textual#1021] [General]
Hey Textual team, probably one of those things that you cannot see once you have seen the checkboxes.  I notices while browsing the docs that there are a number of pages with multiple unchecked checkboxes, but none that are checked.  Having not seen a checked box in the docs yet, these looked checked to me.

![image](https://user-images.githubusercontent.com/22648375/197912930-9c51208c-0bd9-42cd-9a8b-ad015709a9e7.png)

**Comment by davep** [comment]
Perhaps I'm misunderstanding what you're saying, but while it's true that `Bubbles` and `Verbose` aren't "green" for [the Blur event](https://textual.textualize.io/events/blur/), both are for [DescedantBlur](https://textual.textualize.io/events/descendant_blur/), ditto for [DescendantFocus](https://textual.textualize.io/events/descendant_focus/), one is [for Enter](https://textual.textualize.io/events/enter/), and so on.

<img width="278" alt="Screenshot 2022-10-26 at 14 12 54" src="https://user-images.githubusercontent.com/28237/198035043-436e2eb4-4f2c-41bb-a0d2-81191797f9cf.png">

<img width="309" alt="Screenshot 2022-10-26 at 14 13 05" src="https://user-images.githubusercontent.com/28237/198035078-b3e1ad20-2057-46e2-95f9-3ce6fba8780a.png">

<img width="169" alt="Screenshot 2022-10-26 at 14 13 12" src="https://user-images.githubusercontent.com/28237/198035111-66e4132a-6752-4999-b951-b60100c6efda.png">

**Reply by WaylonWalker** [reply]
For someone who has never seen the docs, if they land on the Blur page, it appears checked.  Once you have seen any of the green checks it's easy to see and understand, but if you have never seen the green check and land on a page that is only grey checks, they look checked.

This might be a comment more for material for mkdocs than it is in your setup.  let me know if that is the case.

**Reply by davep** [reply]
I'll check with @willmcgugan when I get the chance, but yeah, [looking at the Material for MkDocs documentation for "task lists"](https://squidfunk.github.io/mkdocs-material/reference/lists/#using-task-lists), it appears to be the styling for those sorts of lists. It does look/feel a wee bit more obvious what's going on if you wish to light mode, but I'd agree it's a bit ambiguous up front.

**Comment by WMRamadan** [comment]
@WaylonWalker This is possible by changing the mkdocs styling. Here is an example of how it can be done (https://github.com/WMRamadan/mkdocs-styling). @davep I would be happy to create an issue and PR for this change.

**Reply by davep** [reply]
That looks great. Yes, please, if you're happy to.
